tidy3d/components/data/monitor_data_n2f.py

```diff
@@ -20,7 +20,11 @@
 class AbstractNear2FarData(MonitorData):
     """Collection of radiation vectors in the frequency domain."""
 
-    monitor: Union[Near2FarAngleMonitor, Near2FarCartesianMonitor, Near2FarKSpaceMonitor]
+    monitor: Union[
```

Comment:
Is this from `black`? I'm running `22.8.0`.  Probably better that this is on one line (and same thing for later down in the file) with `Near2FarDataTypes`

Reply:
Sorry, yeah, in the earlier version when the efficiency was calculated with near2far, I added the `DiffractionEfficiencyMonitor` to this union and split the line. When I removed it later on, I forgot to recombine.